tools/zbfind

```diff
@@ -942,7 +956,13 @@ class ZBPoller(Thread):
 
             # Convert to dBm per AT86RF230 data sheet, figure 5.5:  y=mx+b, b=-91,m=3
             rssi = (3*packetlist[2])-91
-            packet = d154.pktchop(packetlist[0])
+            try:
+                packet = d154.pktchop(packetlist[0])
+            except Exception,e:
+                traceback.print_exc()
+                print "bad packetchop!!"
```

Comment:
Do you have some PCAP of examples where the d154.pktchop failed?
Or would switching to Scapy dot15d4 help avoid this? d154.pktchop is a function from the original KillerBee that was minimum that was needed, but doesn't parse much...